src/Fields/FieldsetRepository.php

```diff
@@ -6,11 +6,13 @@
 use Statamic\Facades\File;
 use Statamic\Facades\Path;
 use Statamic\Facades\YAML;
+use Statamic\Support\Str;
 
 class FieldsetRepository
 {
     protected $fieldsets = [];
     protected $directory;
+    protected $addonDirectories = [];
```

Comment:
Can you simply transform the single `$directory` property to a `$directories` property, so that there is only one prop for directories?

Reply:
I had that originally but ran into an issue, can't remember what it was. I can revisit now that I have everything working.

Reply:
ya can't do that, it's needed